src/plugin/UKPlugin.cpp

```diff
@@ -260,7 +260,7 @@ namespace UKControllerPlugin {
     {
         EuroScopePlugIn::CFlightPlan plan = this->FlightPlanSelect(callsign.c_str());
 
-        if (!plan.IsValid() || plan.GetSimulated()) {
+        if (!plan.IsValid()) {

```

Comment:
Is this stuff meant to be in this change set?

Reply:
It was in a previous release... but apparently has come up on this PR!